image/utilities.sh

```diff
@@ -4,7 +4,7 @@ source /bd_build/buildconfig
 set -x
 
 ## Often used tools.
-$minimal_apt_get_install curl less vim-tiny psmisc
+$minimal_apt_get_install curl less vim-tiny psmisc gpg-agent dirmngr
```

Comment:
I had an image based on the 16.04 phusion image, where 'apt-key add' worked out of the box (no need to install extra packages). That image failed when switching to 18.04 because of these missing dependencies. 

A lot of dockerscripts will start adding external repositories (which require all the key adding and keyserver stuff) so I figured it would be convenient if Dockerfiles would keep working if you switch to 18.04, without having to install extra packages before doing your keyserver thing.

But you might want to wait for other users to weigh in whether they need this dependency or not.

Reply:
That sounds reasonable, we probably want for the upgrade to preserve peoples' experience and not break existing Dockerfiles written with `FROM phusion/baseimage:latest` if possible.

I know that a major concern is the size of the image (and keeping it as small as possible), so I know this question will come up.  Looks like the gpg-agent package itself is less than 1MB installed (however I did not test to see how large the combined set of deps it pulls in with it had cost.)